milabench/validation/nan.py

```diff
@@ -0,0 +1,57 @@
+from collections import defaultdict
+import math
+
+from .validation import ValidationLayer
+
+
+class _Layer(ValidationLayer):
+    """Makes sures the loss we receive is not Nan.
+
+    Notes
+    -----
+    Show a warning if the loss is not decreasing.
```

Comment:
Given that the validator's name is "nan", it's unexpected that it also checks that the loss does not increase. I think it would be better to rename the validator `loss`.

Reply:
done